x/upgrade/abci.go

```diff
@@ -22,10 +22,26 @@ import (
 // skipUpgradeHeightArray is a set of block heights for which the upgrade must be skipped
 func BeginBlocker(k keeper.Keeper, ctx sdk.Context, _ abci.RequestBeginBlock) {
 	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
+
+	var recentPlan string
+	k.IterateDoneUpgrades(ctx, func(name string, _ int64) bool {
```

Comment:
```suggestion
	k.IterateCompletedUpgrades(ctx, func(name string, _ int64) bool {
```

Reply:
May be we can just hav, GetNextOrLastUpgrade or something more suitable, so that we can avoid all this logic here?